operators/nvidia-network-operator/1.1.0/manifests/network-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,423 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "mellanox.com/v1alpha1",
+          "kind": "HostDeviceNetwork",
+          "metadata": {
+            "name": "hostdevicenetwork-sample"
+          },
+          "spec": {
+            "foo": "bar"
+          }
+        },
+        {
+          "apiVersion": "mellanox.com/v1alpha1",
+          "kind": "MacvlanNetwork",
+          "metadata": {
+            "name": "macvlannetwork-sample"
+          },
+          "spec": {
+            "foo": "bar"
+          }
+        },
+        {
+          "apiVersion": "mellanox.com/v1alpha1",
+          "kind": "NicClusterPolicy",
+          "metadata": {
+            "name": "nicclusterpolicy-sample"
+          },
+          "spec": {
+            "foo": "bar"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    operators.operatorframework.io/builder: operator-sdk-v1.7.1+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    description: Deploy and manage NVIDIA networking resources in Kubernetes
+    provider: NVIDIA
+    repository: https://github.com/Mellanox/network-operator/
+    support: NVIDIA
+  name: network-operator.v1.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: HostDeviceNetwork
+      name: hostdevicenetworks.mellanox.com
+      version: v1alpha1
+    - kind: MacvlanNetwork
+      name: macvlannetworks.mellanox.com
+      version: v1alpha1
+    - kind: NetworkAttachmentDefinition
```

Comment:
Again, I don't think you own this CRD, even though the operator may watch the CRs.